connections.py

```diff
@@ -7,6 +7,7 @@
 import time as tm
 import json
 import re
+import glob
 import os.path
```

Comment:
Not needed anymore since we're using glob now

Reply:
Used to check if the file exists